include/alpaka/core/Tuple.hpp

```diff
@@ -11,31 +11,10 @@
 
 #include <alpaka/core/STLTuple/STLTuple.hpp>
 
-#include <cstddef>
-#include <utility>
-
 namespace alpaka
 {
     namespace core
     {
         using namespace ::utility::tuple;
```

Comment:
This is the only meaningful line remaining in this file. Is this still required?

Reply:
I think my change to this file max have been wrong and second commit of the PR has to be removed. But anyhow the highlighted part makes sense to keep imo